### PR TITLE
Mapped product name along with the SKU to the paypal Item object

### PR DESCRIPTION
**Pull request by JudeNiroshan**
Added product name when mapping LineItem to Item in Paypal_plus request.

Addresses: https://github.com/commercetools/commercetools-paypal-plus-integration/issues/140

### CODE REVIEW

File: src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java [hunk 1]
```diff
@@ -253,6 +253,11 @@ protected Item createPaypalPlusItem(@Nonnull LocalizedString itemName, @Nonnull
         return createPaypalPlusItem(itemName, locales, dlipfq.getQuantity(), dlipfq.getDiscountedPrice().getValue());
     }
 
+    private Item plusSkuProductName(@Nonnull LineItem lineItem, @Nonnull List<Locale> locales, @Nonnull Item targetItem) {
+        return targetItem.setSku(lineItem.getVariant().getSku())
+                .setName(lineItem.getName().get(locales));
```
Comment (butenkor) [inline src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java]: Is it ensured that locale is never null?

File: src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java [hunk 2]
```diff
@@ -253,6 +253,11 @@ protected Item createPaypalPlusItem(@Nonnull LocalizedString itemName, @Nonnull
         return createPaypalPlusItem(itemName, locales, dlipfq.getQuantity(), dlipfq.getDiscountedPrice().getValue());
     }
 
+    private Item plusSkuProductName(@Nonnull LineItem lineItem, @Nonnull List<Locale> locales, @Nonnull Item targetItem) {
```
Comment (butenkor) [inline src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java]: Maybe to call variable `pppItem` or `paypalItem` would be better? See naming also here: https://github.com/commercetools/commercetools-paypal-plus-integration/pull/143/files#diff-98208488d399ffeb7c0443fb763ae32aR261

File: src/test/unit/java/com/commercetools/helper/mapper/impl/payment/DefaultPaymentMapperImplTest.java [hunk 3]
```diff
@@ -72,11 +72,11 @@ public void ctpPaymentToPaypalPlus_withDiscount() throws Exception {
                 .isNull();
 
         assertThat(itemList.getItems().size()).isEqualTo(5);
-        assertItem(getItemBySkuQuantityPrice(itemList, "123454323454667", "1", "115.24"), "Halskette", "1", "115.24", "USD");
-        assertItem(getItemBySkuQuantityPrice(itemList, "123454323454667", "3", "115.24"), "Halskette", "3", "115.24", "USD");
+        assertItem(getItemBySkuQuantityPrice(itemList, "123454323454667", "Halskette", "1", "115.24"), "Halskette", "1", "115.24", "USD");
```
Comment (butenkor) [inline src/test/unit/java/com/commercetools/helper/mapper/impl/payment/DefaultPaymentMapperImplTest.java]: Why is `Halskette` defined twice here?

File: src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java (hunk 1 above)
Comment (JudeNiroshan) [reply src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java]: Yes. https://github.com/commercetools/commercetools-paypal-plus-integration/blob/aa0be5dbd8c5d1724ca641ba62582daa4bef1bde/src/main/java/com/commercetools/model/CtpPaymentWithCart.java#L123

File: src/test/unit/java/com/commercetools/helper/mapper/impl/payment/DefaultPaymentMapperImplTest.java (hunk 3 above)
Comment (JudeNiroshan) [reply src/test/unit/java/com/commercetools/helper/mapper/impl/payment/DefaultPaymentMapperImplTest.java]: It is the name of the test Item. so, the string `"Halskette"` has used to equality check

File: src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java (hunk 2 above)
Comment (JudeNiroshan) [reply src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java]: Renamed the variable to `paypalItem`
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/143/commits/0a1b6827d4385c563489ad27efdfb7dee88bc706

**Comment by JudeNiroshan** [comment]
@butenkor Could you please have a look on my new changes?

**Pull request merged**